src/openzeppelin/utils.cairo

```diff
@@ -0,0 +1,13 @@
+use array::ArrayTrait;
+
+#[inline(always)]
+fn check_gas() {
+    match gas::withdraw_gas() {
+        Option::Some(_) => {},
+        Option::None(_) => {
+            let mut data = ArrayTrait::new();
+            data.append('Out of gas');
+            panic(data);
+        },
```

Comment:
When [this commit](https://github.com/starkware-libs/cairo/commit/46dc7d04ab755b06152889e8ed4e8ae27b11a290) makes it into a release of cairo, then these lines should be rewritten as

```suggestion
        Option::None(_) => panic_with_felt252('Out of gas'),
```

Reply:
Also I'm wondering. Would you be able to simply do:

```rust
gas::withdraw_gas().expect('Out of gas');
```